### ISSUE TITLE
Where to specify analytics_label for FCM Options?

### ISSUE THREAD

**Issue by belek** [appleboy/gorush#444]
Hi, how to specify analytics_label parameter in push payload for Firebase Messages?
https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#fcmoptions

**Comment by belek** [comment]
Well, I found out that FCM Options is introduced in new API and gorush uses Legacy API.

**Comment by appleboy** [comment]
@belek Yes, gorush use the [legacy HTTP protocols](https://firebase.google.com/docs/cloud-messaging/send-message#send-messages-using-the-legacy-app-server-protocols) not [v1 HTTP protocol](https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages).

**Comment by belek** [comment]
> @belek Yes, gorush use the [legacy HTTP protocols](https://firebase.google.com/docs/cloud-messaging/send-message#send-messages-using-the-legacy-app-server-protocols) not [v1 HTTP protocol](https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages).

Is there any plans and estimates on migrating to v1 ?

**Comment by appleboy** [comment]
We should move to https://godoc.org/google.golang.org/api/fcm/v1

**Comment by Semior001** [comment]
what about migrating to fcm v1? is it still in plans?

**Comment by appleboy** [comment]
Label associated with the message's analytics data.

```json
{
  "notifications": [
    {
      "tokens": ["token_a", "token_b"],
      "platform": 2,
      "message": "Hello World Android!",
      "title": "You got message",
      "fcm_options": {
        "analytics_label": "example"
      }
    }
  ]
}
```
